classes/submission/SupplementaryFile.inc.php

```diff
@@ -0,0 +1,244 @@
+<?php
+
+/**
+ * @file classes/submission/SupplementaryFile.inc.php
+ *
+ * Copyright (c) 2014-2015 Simon Fraser University Library
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SupplementaryFile
+ * @ingroup submission
+ * @see SubmissionFileDAO
+ *
+ * @brief Supplementary file class.
+ */
+
+import('lib.pkp.classes.submission.SubmissionFile');
+
+class SupplementaryFile extends SubmissionFile {
+	/** @var array image file information */
+	var $_imageInfo;
+
+	/**
+	 * Constructor
+	 */
+	function SupplementaryFile() {
+		parent::SubmissionFile();
+	}
+
+
+	//
+	// Getters and Setters
+	//
+	/**
+	 * Get "localized" creator (if applicable).
```

Comment:
That's Dublin Core terminology. (These are consistent with the OJS 2.x function names, FWIW.)